### PR TITLE
Added NetworkPolicy and Routers section

### PR DESCRIPTION
**Pull request by bfallonf**
From: #6550 

cc @danwinship 

@openshift/team-documentation peer review plz

### CODE REVIEW

File: admin_guide/managing_networking.adoc [hunk 1]
```diff
@@ -988,6 +988,67 @@ to accept any connection allowed by each policy. That is,  connections on any
 port from pods in the *_same_* namespace, and connections on ports `80` and
 `443` from pods in *_any_* namespace.
 
+[[admin-guide-networking-networkpolicy-routers]]
+=== NetworkPolicy and Routers
+
+When using the *ovs-multitenant* plug-in, router traffic is automatically
+allowed into all namespaces, because the routers are normally in the "default"
```
Comment (mburke5678) [inline admin_guide/managing_networking.adoc]: Mark for default? _default_

**Comment by mburke5678** [comment]
One nit. LGTM

**Comment by bfallonf** [comment]
Thanks @mburke5678 . I'll merge.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../admin_guide/managing_networking.adoc#admin-guide-manage-networking[Managing Networking]
|Added the xref:../admin_guide/managing_networking.adoc#admin-guide-networking-networkpolicy-routers[NetworkPolicy and Routers] section.
%

**Pull request merged**